Okta.AspNetCore/OktaMvcOptions.cs

```diff
@@ -59,5 +60,15 @@ public class OktaMvcOptions : AspNet.Abstractions.OktaWebOptions
         /// </summary>
         /// <value>The OnUserInformationReceived event.</value>
         public Func<UserInformationReceivedContext, Task> OnUserInformationReceived { get; set; } = context => Task.CompletedTask;
+
+        /// <summary>
+        /// Gets or sets the event invoked when a failure occurs within the Okta api.
+        /// </summary>
+        public Func<RemoteFailureContext, Task> OnOktaException { get; set; } = context => Task.CompletedTask;
```

Comment:
What do you think if we rename this to `OnOktaApiFailure`? I'm wondering if using just `Okta` is too generic (the whole SDK can be considered Okta). Also, I was looking at the [existing events](https://docs.microsoft.com/ru-ru/dotnet/api/microsoft.aspnetcore.authentication.openidconnect.openidconnectevents?view=aspnetcore-1.0) and this seems to be consistent with the naming that's being used for others.